text/processing.py

```diff
@@ -159,14 +161,16 @@ def tasks(self):
         yield Task("run", rqmt={"mem": 3, "time": 3})
 
     def run(self):
-        self.f_list = " ".join(
+        self.f_list = [
```

Comment:
I now see an issue in this -- the variable is originally different (string instead of lists).
I can change that at an expense of extra memory. But do we even need that variable?

Reply:
I would argue we don't need it as an instanciated list or string. It can also be a generator expression that we use in the loop below.